storage-proofs/porep/src/stacked/vanilla/proof.rs

```diff
@@ -265,7 +271,7 @@ impl<'a, Tree: 'static + MerkleTreeTrait, G: 'static + Hasher> StackedDrg<'a, Tr
         assert!(layers > 0);
 
         // generate labels
-        let (labels, _) = Self::generate_labels(graph, layer_challenges, replica_id, config)?;
```

Comment:
Is `generate_labels` no longer used? If so, should it be removed?

Reply:
it is?

Reply:
should be removed

Reply:
Thanks, I misread the diff.